tutorial/Lab1/README.md

```diff
@@ -0,0 +1,166 @@
+
+   ```console
+   $ cd guestbook/v1
+
+   $ kubectl create -f redis-master-deployment.yaml
+   deployment.apps/redis-master created
+
+   $ kubectl create -f redis-master-service.yaml
+   service/redis-master created
+
+   $ kubectl create -f redis-slave-deployment.yaml
+   deployment.apps/redis-slave created
+
+   $ kubectl create -f redis-slave-service.yaml
+   service/redis-slave created
+
+   $ kubectl create -f guestbook-deployment.yaml
+   deployment.apps/guestbook-v1 created
+
+   $ kubectl create -f guestbook-service.yaml
+   service/guestbook created
+   ```
+   Refer to the [guestbook README](https://github.com/IBM/guestbook) for more details.
+ 
+2. View the guestbook:
+
+   Follow these [steps](https://github.com/IBM/guestbook/tree/master/v1#view-the-guestbook)
+
+Note: We were able to deploy the application by using `kubectl`. However, using `kubectl` here means that you already know the different resource files and the deployment of those resources in a particular order. It would be great if we could simplify this and just concentrate on the app deployment instead.
+
+# Deploy the application using Helm
+
+In this part of the lab, we will deploy the application by using Helm. We will use a release name of `guestbook-demo` to distinguish it from the previous deployment. The chart is available [here](https://github.com/IBM/helm101/tutorial/charts/guestbook).
+
+A chart is defined as a collection of files that describe a related set of Kubernetes resources. We probably then should take a look at the the files before we go and install the chart. The files for the `guestbook` chart are as follows:
+* Chart.yaml: A YAML file containing information about the chart.
+* LICENSE: A plain text file containing the license for the chart.
+* README.md: A README providing information about the chart usage, configuration, installation etc.
+* templates: A directory of templates that will generate valid Kubernetes manifest files when combined with values.yaml. Files contained are as follows:
+   * _helper.tpl: Template helpers/definitions that are re-used throughout the chart.
+   * NOTES.txt: - A plain text file containing short usage notes about how to access the app post install.
+   * guestbook-deployment.yaml: Guestbook app container resource.
+   * guestbook-service.yaml: Guestbook app service resource.
+   * redis-master-deployment.yaml: Redis master container resource.
+   * redis-master-service.yaml: Redis master service resource.
+   * redis-slave-deployment.yaml: Redis slave container resource.
+   * redis-slave-service.yaml: Redis slave service resource.
+* values.yaml: The default configuration values for the chart.
+
+Note: The template files shown above will be rendered into Kubernetes manifest files by Tiller before being passed to the Kubernetes API server. Therefore, they map to the manifest files that we deployed when we used `kubectl` (minus the helper and notes files). 
```

Comment:
@spzala, I agree with what you are saying in principle but changes will need to be done regardless after v3 becomes available as client/server model will no longer apply. I think it is best to write it as is until v3 is released and we can do a PR afterwards. Is that ok?

Reply:
Yup, I agree that changes will require but my intention was to have minimal changes in future as possible since we know it's coming. But either way it looks good and OK with me. Thanks @hickeyma